### ISSUE TITLE
AcceptedCount and Ignored always 0 even though some tracks failed to scrobble.

### ISSUE THREAD

**Issue by coczero** [inflatablefriends/lastfm#92]
I scrobble a lot of tracks at the same time.
Some tracks fail to scrobble, but in the response the AcceptedCount is always 0 aswell as the Ignored list. Is this a bug? I can't seem to find out why some tracks failed to scrobble.

**Comment by coczero** [comment]
Also I just got a "Success" response, but around 2000 tracks failed to scrobble (out of 50000)

**Comment by rikkit** [comment]
Hi @coczero , very sorry for the slow reply. I am currently on holiday so haven't checked GitHub recently.

The Success and AcceptedCount properties are wired up to the JSON response that Last.fm sends. In the past the AcceptedCount property was available in the JSON. I know Last.fm has been changing its JSON output somewhat recently, so maybe this info isn't there anymore. Will need some investigation.

Can you tell me how frequently you tried to scrobble? Maybe you're hitting the upper threshold on scrobbles, documented here: https://github.com/inflatablefriends/lastfm/blob/master/doc/scrobbling.md#there-is-a-limit-on-how-many-scrobbles-may-be-sent-in-each-day . 

> There must be at least 30 seconds in between each scrobble. This means you may scrobble at most (24 * 60 * 60) / 30 = 2880 tracks in one day. If this limit is reached then the scrobbles will be cached if it's enabled.

It used to be that scrobbles that were ignored by this mechanism were reported as ignored, perhaps that has changed as well? 😕

**Comment by coczero** [comment]
Hi,

I am probably way over the upper threshold of the scrobbles, but that value seems to be unpredictable at best. I normally have no problem scrobbling around 10000 tracks within a minute. So maybe the threshold has changed too. The problem is just how can we tell if a scrobble has been rejected and what was the reason?

**Comment by rikkit** [comment]
I haven't had a chance to investigate this yet.

Out of curiosity though why are you scrobbling 10,000 tracks? Stress testing?

**Comment by coczero** [comment]
Basically, I parse the iTunes library to allow people to import their database to a new last.fm account.

**Comment by rikkit** [comment]
@coczero could it be this? https://github.com/inflatablefriends/lastfm/blob/master/doc/scrobbling.md#scrobbles-have-an-age-limit 

There should be better reporting on this either way.

**Comment by coczero** [comment]
No, the scrobbles go through fine. Maybe the api doesnt really give info about this anymore.
